#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_cdr2svg.dpatch by Andreas Wenning <awen@awen.dk>
##
## DP: Patch to use uniconvertor instead of uniconv.
## DP: Makes the .cdr input work again, closes bugs #506217, #471109

@DPATCH@
diff -urNad inkscape-0.46~/share/extensions/cdr2svg.sh inkscape-0.46/share/extensions/cdr2svg.sh
--- inkscape-0.46~/share/extensions/cdr2svg.sh	2008-03-11 05:21:10.000000000 +0100
+++ inkscape-0.46/share/extensions/cdr2svg.sh	2008-05-12 17:13:50.000000000 +0200
@@ -4,7 +4,7 @@
 TEMPFILENAME=`mktemp 2>/dev/null || echo "$TMPDIR/$$"`
 TEMPFILENAME=${TEMPFILENAME}.svg
 
-uniconv "$1" "${TEMPFILENAME}" > /dev/null 2>&1 || rc=1
+/usr/bin/uniconvertor "$1" "${TEMPFILENAME}" > /dev/null 2>&1 || rc=1
 
 cat < "${TEMPFILENAME}" || rc=1
 rm -f "${TEMPFILENAME}"
diff -urNad inkscape-0.46~/share/extensions/cdr_input.inx inkscape-0.46/share/extensions/cdr_input.inx
--- inkscape-0.46~/share/extensions/cdr_input.inx	2008-03-11 05:21:07.000000000 +0100
+++ inkscape-0.46/share/extensions/cdr_input.inx	2008-05-12 17:13:35.000000000 +0200
@@ -1,7 +1,7 @@
 <inkscape-extension>
     <_name>Corel DRAW Input</_name>
     <id>org.inkscape.input.cdr</id>
-    <dependency type="executable" location="path">uniconv</dependency>
+    <dependency type="executable" location="path">uniconvertor</dependency>
     <dependency type="executable" location="extensions">cdr2svg.sh</dependency>
     <input>
         <extension>.cdr</extension>
diff -urNad inkscape-0.46~/share/extensions/wmf_input.inx inkscape-0.46/share/extensions/wmf_input.inx
--- inkscape-0.46~/share/extensions/wmf_input.inx	2008-03-11 05:21:10.000000000 +0100
+++ inkscape-0.46/share/extensions/wmf_input.inx	2008-05-12 17:13:22.000000000 +0200
@@ -1,7 +1,7 @@
 <inkscape-extension>
     <_name>Windows Metafile Input</_name>
     <id>org.inkscape.input.wmf</id>
-    <dependency type="executable" location="path">uniconv</dependency>
+    <dependency type="executable" location="path">uniconvertor</dependency>
     <dependency type="executable" location="extensions">cdr2svg.sh</dependency>
     <input>
         <extension>.wmf</extension>
